### DISCUSSION TITLE
Pokemon Sword and Shield

### DISCUSSION THREAD

**Discussion by Adithecrow** [skyline-emu/skyline#2174] [Q&A] [answered]
The game crashes when you gaigantamax battle gym trainers and it crashes when we try to catch a gaigantamax pokemon, I'm using an Mi 12 pro with the 8 gen 1 chip, is anyone else facing these issues? Or know how to fix them?

**Comment by Crytonics** [comment, marked as answer]
You can ask here https://github.com/skyline-emu/skyline-games-list/issues/248 or in our discord server
